syft/workers/base.py

```diff
@@ -549,23 +551,22 @@ def execute_computation_action(self, action: ComputationAction) -> PointerTensor
                 raise ResponseSignatureError(new_ids)
 
     def execute_communication_action(self, action: CommunicationAction) -> PointerTensor:
```

Comment:
Seems like you don't use the `return_ids` and `return_values` attributes. Do you think we will use them at some point?

Reply:
I think we probably should, but it's not required to make the existing tests pass. As we start tracing more `CommunicationActions`, I think we'll have to fix this, particularly for things like `get`.